templates/frontend/pages/editorialHistory.tpl

```diff
@@ -0,0 +1,53 @@
+{**
+ * templates/frontend/pages/editorialHistory.tpl
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @brief Display context's editorial history page.
+ *
+ *}
+{include file="frontend/components/header.tpl" pageTitle="common.editorialHistory"}
+
+<div class="page page_masthead">
+	{include file="frontend/components/breadcrumbs.tpl" currentTitleKey="common.editorialHistory"}
+
+	<h1>{translate key="common.editorialHistory"}</h1>
+	{foreach from=$mastheadRoles item="mastheadRole"}
+		{if array_key_exists($mastheadRole->getId(), $mastheadUsers)}
+			<h2>{$mastheadRole->getLocalizedName()|escape}</h2>
+			<ul>
+			{foreach from=$mastheadUsers[$mastheadRole->getId()] item="mastheadUser"}
+				{foreach from=$mastheadUser['services'] item="service"}
+					<li>
+						<ul id="commaList">
+							<li>{$mastheadUser['user']->getFullName()|escape}</li>
+							{if !empty($mastheadUser['user']->getLocalizedData('affiliation'))}
+								<li>{$mastheadUser['user']->getLocalizedData('affiliation')|escape}</li>
+							{/if}
+							<li>{$service['dateStart']} - {$service['dateEnd']}</li>
```

Comment:
This should be made a locale key in order to better support right-to-left languages. But maybe that can wait for the markup to be reviewed.